config/application.rb

```diff
@@ -20,7 +20,7 @@
 module PracticalDeveloper
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.1
+    config.load_defaults 5.1 # NOTE: [Rails 6] we should at least work towards updating this to 5.2
```

Comment:
@rhymes Does it cause a lot of problems if we try with 5.2 or 6.0?

Reply:
Yep. if you look at https://github.com/thepracticaldev/dev.to/blob/master/config/initializers/new_framework_defaults_5_2.rb we still have at least one setting disabled. I think we can bump it later on making sure that setting stays disabled for the time being and them bump to 5.2.

I wouldn't bump to 6.0 yet, before we check each setting.

Anyway, I plan to send follow up PRs to address all the remaining deprecations and possibly "Rails-sixify" the code :D